chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/api/JoinedRoomApiTypeMapper.kt

```diff
@@ -0,0 +1,42 @@
+package com.pusher.chatkit.rooms.api
+
+import com.pusher.chatkit.rooms.state.JoinedRoomInternalType
+import com.pusher.chatkit.util.DateApiTypeMapper
+
+internal class JoinedRoomApiTypeMapper {
+    fun toRoomInternalType(room: JoinedRoomApiType): JoinedRoomInternalType {
+
+        var lastMessageAt: Long? = null
+        if (room.lastMessageAt != null) {
+            lastMessageAt = DateApiTypeMapper().mapToEpochTime(room.lastMessageAt)
+        }
```

Comment:
```suggestion
        val lastMessageAt = room.lastMessageAt?.let {
            dateApiTypeMapper.mapToEpochTime(room.lastMessageAt)
        }
```

Reply:
👍 i forgot about the `?.let` thing here! thanks for pointing it out :)